Providers/Profile.swift

```diff
@@ -703,15 +703,22 @@ public class BrowserProfile: Profile {
 
         // Used as a task queue for syncing.
         private var syncReducer: AsyncReducer<EngineResults, EngineTasks>?
+        private var syncStatsReport: SyncStatsReport?
```

Comment:
I'm not a fan of it either. I'll see if I can weave it in to fit in with the more functional code in this module.